examples/java-jfr/run.sh

```diff
@@ -0,0 +1,4 @@
+export PYROSCOPE_APPLICATION_NAME=fibonacci-java-lock-push
+export PYROSCOPE_PROFILER_EVENT=lock
+export PYROSCOPE_SERVER_ADDRESS=http://pyroscope:4040
```

Comment:
I'd move it to docker-compose, close to `PYROSCOPE_LOG_LEVEL` variable. Also would add format explicitly:
```
PYROSCOPE_FORMAT=jfr
```

It's in the [docs](https://pyroscope.io/docs/java/)